### PR TITLE
CF-465 Cinder quota migration fix

### PR DESCRIPTION
**Pull request by roman-verchikov**

### CODE REVIEW

File: cloudferry/lib/os/storage/cinder_storage.py [hunk 1]
```diff
@@ -150,48 +150,74 @@ def _read_info_volumes(self, **kwargs):
...
+            else:
+                filtered_tenants = [self.filter_tenant_id]
+            tenants = [self.identity_client.try_get_tenant_by_id(t) for t in
+                       filtered_tenants]
         else:
-            tenant_ids = \
-                [tenant.id for tenant in
-                 self.identity_client.get_tenants_list()
-                    if tenant.id != service_tenant_id]
+            tenants = [tenant for tenant in
```
Comment (SVilgelm) [inline cloudferry/lib/os/storage/cinder_storage.py]: `tenants = self.identity_client.get_tenants_list()` ???

Comment (roman-verchikov) [inline cloudferry/lib/os/storage/cinder_storage.py]: whoopsie :)

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferry/lib/os/storage/cinder_storage.py [hunk 2]
```diff
@@ -150,48 +150,74 @@ def _read_info_volumes(self, **kwargs):
...
         quotas = []
-        for tenant_id in tenant_ids:
-            quota = self.cinder_client.quotas.get(tenant_id)
-            quota_conv = self.convert_quota(quota)
-            quota_conv['tenant_id'] = tenant_id
-            quota_conv['tenant_name'] = self.identity_client\
-                .try_get_tenant_name_by_id(tenant_id)
-            quotas.append(quota_conv)
+        for t in tenants:
+            quota = self.cinder_client.quotas.get(t.id).to_dict()
```
Comment (SVilgelm) [inline cloudferry/lib/os/storage/cinder_storage.py]: `tenants` can contain `None`, you will get `AttributeError`

Comment (SVilgelm) [inline cloudferry/lib/os/storage/cinder_storage.py]: because `try_get_tenant_by_id` can returns `None`

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferry/lib/os/storage/cinder_storage.py (hunk 2 above)
Comment (SVilgelm) [inline cloudferry/lib/os/storage/cinder_storage.py]: OK. It's fixed

**Comment by SVilgelm** [comment]
👍

**Comment by MirantisWorkloadMobility** [comment]
$PYLINT_STATUS

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
test

**Comment by roman-verchikov** [comment]
retest

**Comment by roman-verchikov** [comment]
retest

**Comment by roman-verchikov** [comment]
retest

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by archyufa** [comment]
+1 tested in staging

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**